monotonic.py

```diff
@@ -165,5 +165,5 @@ def monotonic():
         if monotonic() - monotonic() > 0:
             raise ValueError('monotonic() is not monotonic!')
 
-    except Exception:
-        raise RuntimeError('no suitable implementation for this system')
+    except Exception as e:
+        raise RuntimeError('no suitable implementation for this system : ' + repr(e))
```

Comment:
repr typically includes more information in its output than str. For built-in exceptions str will just return the exception message, while repr will also include the exception class name, which would be useful to me when debugging the issue. Since the error I mentioned above happens infrequently I want to gather as much information about the cause exception as possible.

For example:

```python
import ctypes
import ctypes.util
try:
    ctypes.CDLL(ctypes.util.find_library('c')).missing_function
except Exception as e:
  print 'str', str(e)
  print 'repr', repr(e)
```
outputs

```
/lib/x86_64-linux-gnu/libc.so.6: undefined symbol: missing_function
AttributeError('/lib/x86_64-linux-gnu/libc.so.6: undefined symbol: missing_function', )
```